src/ralph/scan/plugins/ssh_juniper.py

```diff
@@ -45,6 +45,16 @@ def _get_hostname(ssh):
                 pass  # unexpected result... maybe next line will be ok...
 
 
+def _get_model(ssh):
+    for line in _ssh_lines(ssh, 'show version'):
+        if line.lower().startswith('model'):
```

Comment:
Re expression is much better to document required input string, and for future updates 

re.match('model:(.*)', 'model: dupa').groups()